webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -530,13 +544,20 @@ export class SearchableChannelList extends React.PureComponent<Props, State> {
             <div
                 id={'hideJoinedPreferenceCheckbox'}
                 onClick={this.handleChecked}
+                onKeyDown={(e) => {
+                    e.stopPropagation();
+                    if (e.key === 'Enter' || e.key === ' ') {
+                        this.handleChecked();
+                    }
+                }}
+                role='checkbox'
+                aria-checked={this.props.rememberHideJoinedChannelsChecked}
+                aria-label={this.props.intl.formatMessage({id: 'more_channels.hide_joined_channels_checkbox', defaultMessage: 'Hide joined channels checkbox'})}
```

Comment:
Could you also also remove the "checkbox" part? The screen reader already also reads that out because of `role='checkbox'`